File: ArraysACTV/Actv4.cs
//busqueda lineal de un arreglo

class Actv4
{
    public static int BusquedaLineal(int[] arreglo, int valorBuscado)
    {
        for (int i = 0; i < arreglo.Length; i++)
        {
            if (arreglo[i] == valorBuscado)
            {
                return i; // Retorna el índice del valor encontrado
            }
        }
        return -1; // Retorna -1 si el valor no se encuentra en el arreglo
    }
}

File: src/main/java/Arrays/Actv4.java
//busqueda lineal de un arreglo

package Arrays;

public class Actv4 {

    public static void busquedaLineal(int[] arreglo, int valor) {
        boolean encontrado = false;
        for (int i = 0; i < arreglo.length; i++) {
            if (arreglo[i] == valor) {
                System.out.println("Valor " + valor + " encontrado en la posición: " + i);
                encontrado = true;
                break;
            }
        }
        if (!encontrado) {
            System.out.println("Valor " + valor + " no encontrado en el arreglo.");
        }
    }

}
